test/fixtures/suspay.json

```diff
@@ -0,0 +1,59 @@
+{
+  "SUSPAY_CREATE_META_JSON": {
+    "TransactionIndex": 2,
+    "AffectedNodes": [
+      {
+      "CreatedNode": {
+        "LedgerEntryType": "SusPay",
```

Comment:
This is from rippled? It uses `SusPay` ? I thought I recalled @justmoon request the full monty in the json.